app/actions/UserActions.js

```diff
@@ -31,6 +46,25 @@ function removeToken() {
   return cookie.remove(USER_STORAGE_KEY, { path: '/' });
 }
 
+function getToken(getCookie?: string => encodedToken): ?token {
+  const encodedToken = getCookie
+    ? getCookie(USER_STORAGE_KEY)
```

Comment:
might be slightly cleaner (and more explicit) to pass in `cookie.get` as well, so you don't need the conditional here?

Reply:
Agree! Should be good to go now.